### ISSUE TITLE
UI/UX

### ISSUE THREAD

**Issue by ktseo41** [ktseo41/quotes-sharer#31]
템플릿, 글자색, 배경색, 폰트 사이즈, 폰트 패밀리, 로그인, undo, redo 등 다양한 기능을 직관적으로 담고 WYSIWYG 유형의, 부가적인 기능들로 번거로움을 주지 않는 미니멀한 디자인

**Comment by ktseo41** [comment]
https://midjourney.gitbook.io/docs/imagine-parameters#algorithm-modifiers

**Comment by ktseo41** [comment]
### 참조 이미지

1.  https://www.behance.net/gallery/83753221/Note-App-UI-UX
2. https://www.vectorstock.com/royalty-free-vector/text-editor-smartphone-interface-template-mobile-vector-29163981

**Comment by ktseo41** [comment]
> 템플릿, 글자색, 배경색, 폰트 사이즈, 폰트 패밀리, 로그인, undo, redo 등 다양한 기능을 직관적으로 담고 WYSIWYG 유형의, 부가적인 기능들로 번거로움을 주지 않는 미니멀한 디자인

### ChatGPT

One example of a minimalistic design with intuitive functionality is a text editor. Many text editors have a clean and simple interface, while still offering a wide range of features such as templates, text formatting options (like color and font size), undo and redo, and even a login feature for saving and syncing documents. Some popular examples of minimalistic text editors include iA Writer, Bear, and Ulysses.

Another example could be a website or mobile app with a minimalistic design, containing a navigation bar with minimal links such as Home, About, Contact, and Login/Signup. In addition, text color, background color, font size, font family, and other design elements could be controlled through a settings panel accessed via a button or icon in the navigation bar.

You could even think of a simple design of a to-do list application with a clean look, simple layout, and functionality that includes adding, editing, and deleting tasks, setting reminders, and categorizing tasks by tags or project.

In all cases, the key to a minimalistic design with intuitive functionality is to keep the interface uncluttered and easy to navigate, while still providing access to the tools and features that users need to accomplish their tasks.

**Comment by ktseo41** [comment]
### 글쓰기 툴

-  iA Writer, Pure Writer, JotterPad, 스크리브너, SimpleText(https://simpletext.app/)

#### 참조

![image](https://user-images.githubusercontent.com/6429885/211581492-8f17128b-d43c-48ac-a8fc-cbea20877090.png)

https://kingpinstudio.net/%ec%95%84%ec%9d%b4%ed%8c%a8%eb%93%9c-%ea%b8%80%ec%93%b0%ea%b8%b0-%ec%95%b1/

https://seoulalien.substack.com/p/360

**Comment by ktseo41** [comment]
> 템플릿, 글자색, 배경색, 폰트 사이즈, 폰트 패밀리, 로그인, undo, redo 등 다양한 기능을 직관적으로 담고 WYSIWYG 유형의, 부가적인 기능들로 번거로움을 주지 않는 미니멀한 디자인

### MidJourney

![image](https://user-images.githubusercontent.com/6429885/211583157-43498ea8-0b50-4720-b688-d4a6132a29ee.png)

**Comment by ktseo41** [comment]
> 템플릿, 글자색, 배경색, 폰트 사이즈, 폰트 패밀리, 로그인, undo, redo 등 다양한 기능을 직관적으로 담고 WYSIWYG 유형의, 부가적인 기능들로 번거로움을 주지 않는 미니멀한 디자인

### ChatGPT 2

An example of a minimalistic design that intuitively contains various functions such as template, text color, background color, font size, font family, login, undo, redo, etc. could be a web-based word processor.

The interface could have a simple and clean layout, with a toolbar at the top containing buttons for common formatting options such as text color, background color, font size, and font family. A template button could be included in the toolbar as well, allowing users to quickly apply predefined styles and layouts to their documents.

Additionally, the word processor could have a login feature, allowing users to save their documents online and access them from anywhere. The undo and redo buttons could be included in the toolbar, as well as in the right-click context menu or through keyboard shortcuts.

The interface could also include a minimalistic layout, such as a side-panel, which could include the basic formatting options, and a document area.

In short, a minimalistic design that intuitively contains various functions such as template, text color, background color, font size, font family, login, undo, redo etc. should have an interface that is clean and easy to navigate, with buttons for common actions and formatting options easily accessible and grouped in intuitive ways.

**Comment by ktseo41** [comment]
> template for sharting short quote. with author, book title. ui, ux, sns, instgram, 1:1 ratio. minimal style. white theme

![bohyeon_template_for_sharting_short_quote _with_author_book_tit_d3a4796a-4010-45ef-b2f4-4515bff3f273](https://user-images.githubusercontent.com/6429885/212079239-ba8f5ea5-01cc-45e6-b612-b82f40f5a3cd.png)

**Comment by ktseo41** [comment]
> template for sharting short quote. with author, book title. ui, ux, sns, instgram, 1:1 ratio. no layout. clean. simple

![bohyeon_template_for_sharting_short_quote _with_author_book_tit_cbbb17d0-3327-49d1-a1e0-c0799a373942](https://user-images.githubusercontent.com/6429885/212079651-d516d14b-c4d1-4055-bc84-c87e07566dd1.png)

**Comment by ktseo41** [comment]
> template for sharing short quote. with author, book title. ui, ux, sns, instgram, 1:1 ratio. no layout. clean. simple. max detail

![Uploading bohyeon_template_for_sharing_short_quote._with_author_book_titl_85385e11-5bc1-426e-8664-cec24bec5fb8.png…]()

**Comment by ktseo41** [comment]
> template for sharing short quote. with author, book title. ui, ux, sns, instgram, 1:1 ratio. no layout. clean. simple. max detail

![bohyeon_template_for_sharing_short_quote _with_author_book_titl_85385e11-5bc1-426e-8664-cec24bec5fb8](https://user-images.githubusercontent.com/6429885/212080436-089a8780-360e-4d7b-9711-f97635f326d3.png)

**Comment by ktseo41** [comment]
> template for sharing short quote. with author, book title. ui, ux, sns, instgram, 1:1 ratio. super simple. max detail. white text color. dark blue background color

![Uploading bohyeon_template_for_sharing_short_quote._with_author_book_titl_663398e1-91fe-4640-b905-3d09f61ca7d6.png…]()

**Comment by ktseo41** [comment]
> template for sharing quote, title, author. ui ux, high quality, max detail. modern design

![bohyeon_template_for_sharing_quote_title_author _ui_ux_high_qua_78851551-ca62-4c24-bc3c-29a10b9f5fbd](https://user-images.githubusercontent.com/6429885/212211325-41ff9ada-e598-41dd-8da0-ea9e8201cae8.png)

**Comment by ktseo41** [comment]
> image template for sharing sentences, title, author. ui ux, high quality, max detail. modern design. no frame

![bohyeon_image_template_for_sharing_sentences_title_author _ui_u_a635a1a8-eaff-4578-98e7-5063c7df37bf](https://user-images.githubusercontent.com/6429885/212211387-2c8d4556-5c28-413b-ada7-03884a9a5b99.png)

**Comment by ktseo41** [comment]
> image template for sharing short sentences, title, author. ui ux, high quality, max detail. cyberpunk style. no frame

![bohyeon_image_template_for_sharing_short_sentences_title_author_a1abb988-a3e5-4fe9-b936-634b189b9daf](https://user-images.githubusercontent.com/6429885/212211621-75f17b67-3bec-4c67-85f8-656110013140.png)

**Comment by ktseo41** [comment]
> image template for sharing short sentences, title, author. ui ux, no background image, pixelart style. no frame

![bohyeon_image_template_for_sharing_short_sentences_title_author_54cbe691-11d4-4b2e-a901-bcdefebdace2](https://user-images.githubusercontent.com/6429885/212212151-fb024f4a-bf53-46b8-91c2-d02f7043f252.png)

**Comment by ktseo41** [comment]
> image template for sharing short sentences, title, author. ui ux, pixelart style. no frame, no background image, no image.

![bohyeon_image_template_for_sharing_short_sentences_title_author_8e49feea-7ebb-4b01-8c00-f97a6ea79334](https://user-images.githubusercontent.com/6429885/212212932-c6d6dc8c-766a-45a4-adde-71e96e818edd.png)

**Comment by ktseo41** [comment]
> image template for sharing short sentences, title, author. ui ux, pixelart style. very simple. no frame, no background image, no image

![bohyeon_image_template_for_sharing_short_sentences_title_author_7606a63b-2c2d-4807-8ff7-17fd57534e1d](https://user-images.githubusercontent.com/6429885/212213164-3369e54b-d3bd-4427-b850-a0eb999b42f1.png)

**Comment by ktseo41** [comment]
## 책 이미지 픽셀아트화

https://giventofly.github.io/pixelit/#tryit

![9791196617141](https://user-images.githubusercontent.com/6429885/212213291-166321fe-4fb1-4fee-9d84-89edb976c78a.jpg)
![pxArt](https://user-images.githubusercontent.com/6429885/212213295-d58c7e3d-f5af-4328-909d-61c8e59a3d73.png)

**Comment by ktseo41** [comment]
> image template for sharing short sentences, title, author. ui ux, pixelart style. very simple. no frame, no background image, no image. no characters

![bohyeon_image_template_for_sharing_short_sentences_title_author_8f13e749-8275-4af7-8ded-1bf9c48f7e11](https://user-images.githubusercontent.com/6429885/212213369-9b5b1bc1-9e0c-4c8e-b30b-34e62fc04ba5.png)

**Comment by ktseo41** [comment]
## 도트 폰트

https://thinking-pingu.tistory.com/104

**Comment by ktseo41** [comment]
> template for sharing short sentences, title, author. ui ux, pixel 8bit style. very simple. no frame, no background image, no image

![bohyeon_template_for_sharing_short_sentences_title_author _ui_u_b13412a7-2627-466b-aa83-910546b25ab4](https://user-images.githubusercontent.com/6429885/212213989-39bf46ad-945f-44c9-b6b1-43fc793c97b9.png)

**Comment by ktseo41** [comment]
> template for sharting short sentences with book title, author name. simple. sans font. ui ux. award winning

![bohyeon_template_for_sharting_short_sentences_with_book_title_a_a0703a77-6fe3-4bf8-befb-ba1298773f22](https://user-images.githubusercontent.com/6429885/212220970-32746427-47b1-44f5-9904-cad97351b379.png)

**Comment by ktseo41** [comment]
> template for sharing short sentence with book title, author name. simple. sans font. ui ux. pixelated

![image](https://user-images.githubusercontent.com/6429885/212223354-e0fc4c3d-ccb8-4ee5-9051-1e4079935cf5.png)

**Comment by ktseo41** [comment]
> sharing short sentence with book title, author name. simple. sans font. ui ux. pixelated. no background image

![bohyeon_sharing_short_sentence_with_book_title_author_name _sim_38e4586d-9ca5-4fa6-98c6-7400766d0a45](https://user-images.githubusercontent.com/6429885/212223761-c1a3892d-59ca-4536-967d-a0bbd999c682.png)

**Comment by ktseo41** [comment]
> sharing short sentence with book title, author name. simple. sans font. ui ux. pixelated. 8 bit color

![bohyeon_sharing_short_sentence_with_book_title_author_name _sim_e010b1f9-677f-4c4d-bb92-6e7ae0ee470e](https://user-images.githubusercontent.com/6429885/212224187-1cafab6a-989b-4d1e-a67f-00c22471f1c2.png)

**Comment by ktseo41** [comment]
![image](https://user-images.githubusercontent.com/6429885/213704718-aedb1757-52d4-47e7-b7fe-5f7ef0c88ccf.png)

**Comment by ktseo41** [comment]
![image](https://user-images.githubusercontent.com/6429885/218496072-52d70fa8-4427-45af-b270-da7d1f1ba84f.png)

**Comment by ktseo41** [comment]
https://wiredjs.com/

**Comment by ktseo41** [comment]
https://maxbittker.github.io/broider/

**Comment by ktseo41** [comment]
http://polyducks.co.uk/what-is-textmode/
